library/OrderCloud.Catalyst/Extensions/ExtensionMethods.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OrderCloud.Catalyst.Extensions
+{
+    public static class ExtensionMethods
```

Comment:
If this is not being used anymore it can be axed

Reply:
IsNullOrEmpty is being used in `CatalystBaseException` line `33` to determine if a List of Errors is **not** null or empty.  I could ax it though, and then just axe this check in the BaseException file?